api/tests/acceptance/application/session/session-controller-enroll-students_test.js

```diff
@@ -120,11 +124,14 @@ describe('Acceptance | Controller | session-controller-enroll-students-to-sessio
                 'last-name': student.lastName,
                 'birth-country': null,
                 'birth-province-code': null,
+                'birth-insee-code': birthCityCode,
+                'birth-postal-code': null,
                 'email': null,
                 'external-id': null,
                 'extra-time-percentage': null,
                 'result-recipient-email': null,
                 'schooling-registration-id': student.id,
+                'sex': 'M',
```

Comment:
Ca je ne comprends pas du tout. Les informations CPF provenant de la schooling-registration copiées dans le certification-candidate ne viennent pas du front !
Elles sont copiées dans le usecase au moment où on récupère la schooling-registration :
api/lib/domain/usecases/enroll-students-to-session.js:29

Reply:
Ça m'a l'air correct, on met un schooling registration avec ces données dans la base, et on vérifie qu'elles sont renvoyées par le PUT (discussion) ?

Reply:
Proposition pour ressembler aux autres champs :
```suggestion
                'birth-insee-code': student.birthCityCode,
                'birth-postal-code': null,
                'email': null,
                'external-id': null,
                'extra-time-percentage': null,
                'result-recipient-email': null,
                'schooling-registration-id': student.id,
                'sex': student.sex,
```